Compiler error after updating pico-sdk

After updating my installation of the pico-sdk to the latest (2.1.0)  I get this error in the examples:

```
[  0%] Built target bs2_default
[  0%] Built target bs2_default_library
[ 20%] Built target pico_lorawan_default_dev_eui
[ 40%] Built target pico_lorawan_erase_nvm
[ 60%] Built target pico_lorawan_hello_abp
[ 80%] Built target pico_lorawan_hello_otaa
[ 80%] Building C object examples/otaa_temperature_led/CMakeFiles/pico_lorawan_otaa_temperature_led.dir/main.c.o
/home/jerryneedell/projects/pico-lorawan/examples/otaa_temperature_led/main.c:28:17: error: 'PICO_DEFAULT_SPI_INSTANCE' undeclared here (not in a function)
   28 |         .inst = PICO_DEFAULT_SPI_INSTANCE,
      |                 ^~~~~~~~~~~~~~~~~~~~~~~~~
make[2]: *** [examples/otaa_temperature_led/CMakeFiles/pico_lorawan_otaa_temperature_led.dir/build.make:76: examples/otaa_temperature_led/CMakeFiles/pico_lorawan_otaa_temperature_led.dir/main.c.o] Error 1
make[1]: *** [CMakeFiles/Makefile2:2104: examples/otaa_temperature_led/CMakeFiles/pico_lorawan_otaa_temperature_led.dir/all] Error 2
make: *** [Makefile:91: all] Error 2
```

The error can be  resolved by changing 
```
         .inst = PICO_DEFAULT_SPI_INSTANCE,
```
to
```
         .inst = PICO_DEFAULT_SPI_INSTANCE(),
```
With this change the example builds and runs as expected.

This appears to be a known breaking change in the SDK
https://github.com/raspberrypi/pico-sdk/issues/1804#issuecomment-2284575940

The change is needed in all the examples.

Can anyone confirm/deny this behavior? I'll be happy to submit a PR for it, but I want to make sure I have not misunderstood something.

--- a/examples/erase_nvm/main.c
+++ b/examples/erase_nvm/main.c
@@ -18,7 +18,7 @@
 // pin configuration for SX1276 radio module
 const struct lorawan_sx1276_settings sx1276_settings = {
     .spi = {
-        .inst = PICO_DEFAULT_SPI_INSTANCE,
+        .inst = PICO_DEFAULT_SPI_INSTANCE(),
         .mosi = PICO_DEFAULT_SPI_TX_PIN,
         .miso = PICO_DEFAULT_SPI_RX_PIN,
         .sck  = PICO_DEFAULT_SPI_SCK_PIN,

--- a/examples/hello_abp/main.c
+++ b/examples/hello_abp/main.c
@@ -22,7 +22,7 @@
 // pin configuration for SX1276 radio module
 const struct lorawan_sx1276_settings sx1276_settings = {
     .spi = {
-        .inst = PICO_DEFAULT_SPI_INSTANCE,
+        .inst = PICO_DEFAULT_SPI_INSTANCE(),
         .mosi = PICO_DEFAULT_SPI_TX_PIN,
         .miso = PICO_DEFAULT_SPI_RX_PIN,
         .sck  = PICO_DEFAULT_SPI_SCK_PIN,

--- a/examples/hello_otaa/main.c
+++ b/examples/hello_otaa/main.c
@@ -21,7 +21,7 @@
 // pin configuration for SX1276 radio module
 const struct lorawan_sx1276_settings sx1276_settings = {
     .spi = {
-        .inst = PICO_DEFAULT_SPI_INSTANCE,
+        .inst = PICO_DEFAULT_SPI_INSTANCE(),
         .mosi = PICO_DEFAULT_SPI_TX_PIN,
         .miso = PICO_DEFAULT_SPI_RX_PIN,
         .sck  = PICO_DEFAULT_SPI_SCK_PIN,

--- a/examples/otaa_temperature_led/main.c
+++ b/examples/otaa_temperature_led/main.c
@@ -25,7 +25,7 @@
 // pin configuration for SX1276 radio module
 const struct lorawan_sx1276_settings sx1276_settings = {
     .spi = {
-        .inst = PICO_DEFAULT_SPI_INSTANCE,
+        .inst = PICO_DEFAULT_SPI_INSTANCE(),
         .mosi = PICO_DEFAULT_SPI_TX_PIN,
         .miso = PICO_DEFAULT_SPI_RX_PIN,
         .sck  = PICO_DEFAULT_SPI_SCK_PIN,

